### PR TITLE
Nitpick docs

### PR DESCRIPTION
**Pull request by roj1512**
- Consistently refer to Node.js as Node.js, not Node.
- Other minor improvements.

### CODE REVIEW

**Review by ry — COMMENTED** [review]
I mean, if that helps readers. But the name of project is "Node".

**Comment by roj1512** [comment]
@ry It is referred to Node.js in their docs, and you already have a number of mentions of it as _Node.js_. Here’s an example:

https://github.com/jsr-io/jsr/blob/9fd9987661488134989623578d69c8955ee1dfe2/frontend/docs/npm-compatibility.md?plain=1#L7

Regardless of which one is right, my goal is to make it consistent. Do you want to switch to _Node_?

File: frontend/docs/badges.md [hunk 1]
```diff
@@ -1,63 +1,62 @@
 ---
-description: JSR provides a badges for each package that can be used to showcase the package version or JSR score for a package.
+description: JSR provides badges for each package that can be used to showcase its latest version or its score.
 ---
 
-For every package, JSR provides an SVG badge that can be included in READMEs,
-documentation, or other places to showcase the package version or JSR score for
-a package.
+JSR provides badges for each package that can be used to showcase its latest
+version or its score.
```
Comment (lucacasonato) [inline frontend/docs/badges.md:6]:
```suggestion
version or its JSR score.
```

File: frontend/docs/badges.md [hunk 2]
```diff
@@ -1,63 +1,62 @@
 ---
-description: JSR provides a badges for each package that can be used to showcase the package version or JSR score for a package.
+description: JSR provides badges for each package that can be used to showcase its latest version or its score.
```
Comment (lucacasonato) [inline frontend/docs/badges.md:2]:
```suggestion
description: JSR provides badges for each package that can be used to showcase its latest version or its JSR score.
```

File: frontend/docs/badges.md [hunk 3]
```diff
@@ -1,63 +1,62 @@
...
+[![JSR Score](https://jsr.io/badges/@<scope>/<package>/score)](https://jsr.io/@<scope>/<package>)
 ```
 
-In a HTML document, use the following code:
+In HTML documents, the following can be used:
 
 ```html
 <a href="https://jsr.io/@<scope>/<package>">
-  <img src="https://jsr.io/badges/@<scope>/<package>/score" alt="">
+  <img src="https://jsr.io/badges/@<scope>/<package>/score" alt="JSR Score" />
```
Comment (lucacasonato) [inline frontend/docs/badges.md:60]: The lack of `alt` was intentional - we can provide a better alt text in the svg itself. Please revert.

Comment (roj1512) [reply frontend/docs/badges.md:60]: Done, but I don’t understand. How can the alt in the SVG be effective if the SVG itself fails to load?

Comment (lucacasonato) [reply frontend/docs/badges.md]: That is not what alts are for. Alt is the alternate text for screen readers.

Comment (roj1512) [reply frontend/docs/badges.md]: Alright.

**Review by lucacasonato — APPROVED** [review]
Thank you

**Pull request merged**